src/ajax.js

```diff
@@ -333,7 +349,15 @@ jQuery.extend({
 			"text html": true,
 
 			// Evaluate text as a json expression
-			"text json": jQuery.parseJSON,
+			"text json": function(json) {
```

Comment:
Totally agree, this was to see if I could get the tests passing and nothing more. The actual implementation will be this: `return (new Response(string)).json();`